### DISCUSSION TITLE
Shutdown mode

### DISCUSSION THREAD

**Discussion by rbbarber-g** [NatLabRockies/ROSCO#229] [Q&A]
I'd like to model the dynamics around cutout, ie when the turbine is moving in and out of its shutdown position, but I can't find any documentation about how the ROSCO shutdown mode works. Does it just feather off the blades once the filtered blade pitch signal reaches SD_MaxPit? If the blades are individually controlled, will it react to any blade? I saw a comment that the turbine can't restart after a shutdown -- is that true? It seems... unrealistic.

If this information is published somewhere I'd be very glad to be pointed in that direction. Thanks for your help.

**Comment by dzalkind** [comment]
Hi,

The shutdown logic can be found here: https://github.com/NREL/ROSCO/blob/41f2c0015dbdeac1307ba798d7590c61a40bd11e/ROSCO/src/ControllerBlocks.f90#L383 

The collective pitch angle triggers a shutdown. Once it's shut down, it stays shut down.  If you have suggestions for logic updates, please share them or contribute them to the code.

A soft cut-out could be implemented with code from #170, but it's not fully supported yet.

Best, Dan

**Comment by mlidGWT** [comment]
I've been reviewing some of the discussions posted, and I noticed that another was turned into an Issue given the development efforts to support all DLCs (discussion #317 ). I would suggest that having the proper wind speed shut down/restart behavior is a critical component for system loads and that the 61400-1/-3 standard highlights many DLCs where simulating at V_out is required for that very reason. Unfortunately, I have nothing to offer here other than highlighting the need for such controller logic.
